corehq/apps/repeaters/models.py

```diff
@@ -50,7 +50,7 @@
 def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False, *args, **kwargs):
     # no control characters (e.g. '/') in keys
     key = hashlib.md5(
-        '{0} timeout {1}'.format(__name__, url)
+        u'{0} timeout {1} {2}'.format(__name__, url, data).encode('utf-8')
```

Comment:
I offlined with @proteusvacuum, but basically we agreed to just cut those two things out entirely (the pummeling and the error caching), and replace the error caching with a datadog counter of timeouts so that the next there's a pileup in the queue, we can check the timeouts graph to see if there's been a spike in timeouts to rule that out or identify it as the cause.